packages/actions/index.ts

```diff
@@ -462,7 +466,9 @@ export const getFriendRequests = () => {
   return async (dispatch, getState) => {
     const user = getUser(getState())()
     const rawPendingFriends = await creditProtocol.getFriendRequests(user.address)
-    const pendingFriends = rawPendingFriends.map(jsonToPendingFriend)
+    const pendingFriends = rawPendingFriends.map(jsonToPendingFriend).sort( (friend1, friend2) => {
+      return friend1.nickname.localeCompare(friend2.nickname)
```

Comment:
we can add the user's language in here so that it sorts for all languages. We just need to import it on line 30 and change the current `language` const to `languageValues` or something.

Example: change line 30 to: `import languageValues, { language } from 'language'`
Then change line 31 to: `const { accountManagement, debtManagement, settlementManagement, copiedClipboard, lndrVerified } = languageValues`

Then pass `language` as the second parameter of `localeCompare`